tools/util_test.py

```diff
@@ -82,8 +82,9 @@ def parse_unit_test_output_test():
 def parse_wrk_output_test():
     print "Testing util.parse_wrk_output_test()..."
     f = open(os.path.join(util.root_path, "tools/testdata/wrk1.txt"))
```

Comment:
It would be cool to test another example where the output used units other than "ms"

Maybe you can use this as wrk2.txt:

```
Running 10s test @ http://127.0.0.1:4544/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   402.90us    1.15ms  1.25us   94.86%
    Req/Sec    26.86k     2.01k   31.81k    78.71%
  539721 requests in 10.10s, 26.25MB read
Requests/sec:  53435.75
Transfer/sec:      2.60MB
```